recipes/http-classes-with-new/src/workflow.ts

```diff
@@ -0,0 +1,72 @@
+import { getNodeImportStatements } from '@nodejs/codemod-utils/ast-grep/import-statement';
+import { getNodeRequireCalls } from '@nodejs/codemod-utils/ast-grep/require-call';
+import { resolveBindingPath } from '@nodejs/codemod-utils/ast-grep/resolve-binding-path';
+import type { SgRoot, Edit, SgNode } from '@codemod.com/jssg-types/main';
+
+const classNames = [
+	'Agent',
+	'ClientRequest',
+	'IncomingMessage',
+	'OutgoingMessage',
+	'Server',
+	'ServerResponse',
+];
+
+/**
+ * Transform function that converts deprecated node:http classes to use the `new` keyword
+ *
+ * Handles:
+ * 1. http.Agent() -> new http.Agent()
+ * 2. http.ClientRequest() -> new http.ClientRequest()
+ * 3. http.IncomingMessage() -> new http.IncomingMessage()
+ * 4. http.OutgoingMessage() -> new http.OutgoingMessage()
+ * 5. http.Server() -> new http.Server()
+ * 6. http.ServerResponse() -> new http.ServerResponse()
```

Comment:
```suggestion
 * 1. `http.Agent()` → `new http.Agent()`
 * 2. `http.ClientRequest()` → `new http.ClientRequest()`
 * 3. `http.IncomingMessage()` → `new http.IncomingMessage()`
 * 4. `http.OutgoingMessage()` → `new http.OutgoingMessage()`
 * 5. `http.Server()` → `new http.Server()`
 * 6. `http.ServerResponse() → `new http.ServerResponse()`
```

Backtick allow readers to know what is code and what is comment.